sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/eventsourcedentity/ReflectiveEventSourcedEntityRouter.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.springsdk.impl.eventsourcedentity
+
+import com.google.protobuf.any.{ Any => ScalaPbAny }
+import kalix.javasdk.eventsourcedentity.{ CommandContext, EventSourcedEntity }
+import kalix.javasdk.impl.eventsourcedentity.EventSourcedEntityRouter
+import kalix.springsdk.impl.{ ComponentMethod, InvocationContext }
+
+import java.lang.reflect.Method
+
+class ReflectiveEventSourcedEntityRouter[S, E <: EventSourcedEntity[S]](
+    override protected val entity: E,
+    componentMethods: Map[String, ComponentMethod],
+    handlerMethods: Map[Class[_], Method])
```

Comment:
Maybe better if we call them `commandHandlerMethods` and `eventHandlerMethods`. 

Up to now we only had examples of `ComponentMethod`, but in reality they were all command handlers. 

Anyway, not that important. We can change this on another PR eventually.